src/index.html

```diff
@@ -32,55 +32,73 @@ <h2>You've seen
         <h2>get</h2>
         <h3>current trips</h3>
       </div>
-      <button type="button" name="go-to" value="go-to" aria-label="current trips" class="go-to go-to-current-js"> <img src="./images/arrow.png" alt="go to current trips"></button>
+      <button type="button" name="go-to" value="go-to" aria-label="current trips" class="go-to go-to-current-js"> <img src="./images/arrow.png" alt="arrow"></button>
```

Comment:
Nice alts on the button images, those are easy to miss! (read: I need to add those!)